.buildbot_dockerfile_debian

```diff
@@ -0,0 +1,17 @@
+FROM debian:bullseye
+
+WORKDIR /ci
+
+RUN apt-get update && apt-get -y install curl build-essential
+
+ENV PATH="${HOME}/.cargo/bin/:${PATH}"
+ENV CARGO_HOME="${HOME}/.cargo"
+ENV RUSTUP_HOME="${HOME}/.rustup"
+
+# Installing Rust
+RUN curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs > rustup.sh
+RUN sh rustup.sh --default-host x86_64-unknown-linux-gnu --default-toolchain stable -y --no-modify-path
```

Comment:
We don't normally do it like this. I'm not super against it though.

Questions:
 - How do you ensure the rust tools will be in $`PATH`?
 - Are the `RUN` lines running as root? The tests will run as a different user.
 - Should the two `RUN` lines be concatenated with a `&&` to prevent the first line being cached, but not the second? Maybe it's ok? https://docs.docker.com/develop/develop-images/dockerfile_best-practices/#run

Reply:
About the user stuff, it may help to take a look at the script that invokes docker.

https://github.com/softdevteam/buildbot_config/blob/master/bin/run_docker_ci_job

`CI_UID` is the UID of the normal user to run the CI job as.

Reply:
PATH is set in the same file:
```
ENV PATH="${HOME}/.cargo/bin/:${PATH}"
ENV CARGO_HOME="${HOME}/.cargo"
ENV RUSTUP_HOME="${HOME}/.rustup"
```

Reply:
Right, but isn't that root's home when building the image, and `CI_UID`'s home when running the job?

Reply:
good question, 
I think this docker will be run as root anyway unless we set: `USER $CI_UID`? which is not set in the [example](https://github.com/softdevteam/lang_tester/blob/a78e4bc037ec300a1767dba587c3d44bea86e009/.buildbot_dockerfile_debian)

also the [buiild](https://ci.soft-dev.org/#/builders/1/builds/1968) passed

Reply:
im going to revert it back to as it was in the provided examples for consistency's sake :)